### PR TITLE
#739 Inspect API - Fixing duplicated keywords + themes in CatalogMetadata

### PR DESCRIPTION
**Pull request by nimshi89**
Changed the select_catalog_metadata.sparql to return distinct values of themes, keyword and landing pages.

Satisfies ticket #739.

### CODE REVIEW

**Review by NickPapONS — COMMENTED** [review]
Looks good! The change to the sparql query makes sense and I like seeing the whole team's work on the cached properties come into effect by making that unit test simpler/more efficient.

File: tests/unit/utils/sparqlhandler/test_csvw_inspector.py [hunk 1]
```diff
@@ -256,3 +256,34 @@ def test_get_column_definitions_for_csv():
         value_url="http://example.com/value",
         virtual=True,
     )
+
+
+def test_multi_theme_and_keyword():
+    """
+    This test ensures that the 'select_catalog_metadata.sparql' will return the correct amount of
+    themes, keywords and landing pages(unique values and no duplications)
```
Comment (robons) [inline tests/unit/utils/sparqlhandler/test_csvw_inspector.py:264]: It'd be helpful if you could stick the URL to the bug in here.

**Review by robons — APPROVED** [review]
Looks good.

It might be helpful to alter the test case you've got to artificially add more themes (and some landing pages).

File: tests/unit/utils/sparqlhandler/test_csvw_inspector.py (hunk 1 above)
Comment (nimshi89) [reply tests/unit/utils/sparqlhandler/test_csvw_inspector.py:264]: yeah sure

**Pull request merged**